src/tracker.h

```diff
@@ -30,13 +30,55 @@
 #ifndef SSR_TRACKER_H
 #define SSR_TRACKER_H
 
+#include <atomic>
+#include <thread>
+
+
+namespace ssr
+{
+
 /// Class definition
-struct Tracker
+class Tracker
 {
-  virtual ~Tracker() = default;  ///< destructor
+  public:
+    virtual ~Tracker() = default;  ///< destructor
+
+    /// calibrate tracker; set the instantaneous position to be the reference
+    virtual void calibrate() = 0;
```

Comment:
I don't think "calibrate" is the right word for this. What about renaming it to something like "reset"?